crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -785,11 +799,13 @@ mod test {
         // https://github.com/software-mansion/starknet.py/blob/cea191679cbdd2726ca7989f3a7662dee6ea43ca/starknet_py/tests/e2e/docs/guide/test_cairo1_contract.py#L29-L36
         let cases = [
             (
-                "0x5167b09ea07d236371efa6053537eb2e6163bdb531e7161643a7c1fddfc5814",
+                // TODO verify calculation of this
```

Comment:
TODO

Reply:
I think we should use the linked python code to verify that and remove the TODOs. You can create an issue and we can verify in separate PR.